setup.py

```diff
@@ -22,7 +22,8 @@
     "pyyaml",
     "cloudpickle>=1.6.0,<=2.2.1,!=2.1.0,!=2.2.0;python_version<'3.11'",
     "cloudpickle==2.2.1;python_version~='3.11'",  # backend only supports cloudpickle 2.2.1 + python 3.11 at the moment
-    "protobuf>=5.28",  # Snowpark IR
+    # `protoc` < 3.20 is not able to generate protobuf code compatible with protobuf >= 3.20.
```

Comment:
Consider deleting this line, or simplify the comment to say `Protobuf 3.20 introduced changes incompatible with older versions`.